pylint/extensions/nested_min_max.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
I think this can become part of `stdlib.py`. I don't think it needs to be an extension.

Reply:
Does it mean it will be activated by default?

Reply:
Yes!

Reply:
I think it would be better if we keep the file as it is before adding to stdlib (I think the base checker is an example of a checker dispathed across multiple files, the checker name is the same in all files).

Reply:
Doesn't the splitting across multiple files make `pylint` slower? There are more functional calls, class initialisations etc. If the file isn't too large we can combine them right?

Reply:
It make everything harder to maintain though. Clearly differentiating checks help tremendously when keeping clean code because it's possible to understand everything about a 50 lines checker, but it's clearly impossible to understand and thus cleanup a 2000 lines stdlib checker behemoth if a function, constant (or more interestingly for performances: condition or calculation) become useless.

Reply:
Hmm, okay. So how would you want to add this then?

Reply:
If we want to make this a default check we can add a new checker (just move the file in pylint/checker) and make it a default checker (register it) with a new name or with an existing checker name.

Reply:
Changed it into a new checker!
Let me know which existing checker name I should use if you have any preference